tests/hopp/test_hybrid.py

```diff
@@ -36,11 +48,55 @@ def site():
                                                                 grid_aspect_power=0.5,
                                                                 row_phase_offset=0.5)
                 },
+                'wave':{
+                    'device_rating_kw': 286, 
+                    'num_devices': 100, 
+                    'wave_power_matrix': [
+                                [0.000000, 0.500000, 1.500000, 2.500000, 3.500000, 4.500000, 5.500000, 6.500000, 7.500000, 8.500000, 9.500000, 10.500000, 11.500000, 12.500000, 13.500000, 14.500000, 15.500000, 16.500000, 17.500000, 18.500000, 19.500000, 20.500000],
```

Comment:
Can you put this power matrix in a separate file and then read it in for `test_hybrid` and `test_wave`?